dldashboard/web/README.md

```diff
@@ -1,54 +1,80 @@
 # Dashboard app
 
-- Install app: `npm install -g yarn ; yarn upgrade`
-- Start app: `yarn start`
-- Start storybook to see individual react components: `yarn storybook`
-- Tests: `yarn test`
-- Add package to project: `yarn add [package name]`
-- Remove package from project: `yarn remove [package name]`
-
-If you get an error on `yarn test` that says `jest-haste-map: Watchman crawl failed. Retrying once with node crawler.`, do `sudo mv /usr/local/bin/watchman /usr/local/bin/watchman.bak`
-
-## create-react-app Readme verbatim below
-
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Available Scripts
+## Know-hows of the app
 
-In the project directory, you can run:
+- To install the app first, do: 
+```
+npm install -g yarn ; yarn upgrade
+```
 
-### `yarn start`
+- Now to start the app, run: `yarn start`.
```

Comment:
should we also mentioned that all the following commands should be ran from `dldashboard/web` folder?

Reply:
Yes, will do. Thank you!